Tests/SwiftSyntaxMacroExpansionTest/MacroSystemTests.swift

```diff
@@ -773,6 +726,21 @@ final class MacroSystemTests: XCTestCase {
     )
   }
 
+  func testCommentsOnExpressionMacro() {
+    assertMacroExpansion(
+      """
+      let b = 
+      /*leading */ #stringify(x + y) /*trailing*/
+      """,
+      expandedSource: """
+        let b = 
+        (x + y, "x + y")
```

Comment:
Do we pass docs in for freestanding expressions?

Reply:
Yes, we do. The macro should re-emit the trivia if it wanted to. The rule is that we strip trivia from freestanding macros and keep trivia of attached macros.